djangocms_versioning/admin.py

```diff
@@ -123,6 +151,13 @@ def get_queryset(self, request):
     def get_changelist(self, request, **kwargs):
         return VersionChangeList
 
+    def get_list_filter(self, request):
+        versionable = versionables.for_content(self.model._source_model)
+        return (
+            fake_filter_factory(versionable, field)
```

Comment:
Why `fake` in the name?
Also, if I wanted to filter by `created_by` too, how would I go about it?

Reply:
Because it's not really filtering, docstring of `fake_filter_factory` says so.
I guess just add `('created_by', RelatedOnlyFieldListFilter)` there.

Reply:
Is there any documentation or explanation what `extra_grouping_fields` setting means? Or even better, or VersionableItem properties?